### PR TITLE
prepare for toolbars/buttons changes

### PR DESCRIPTION
**Pull request by JulianWielga**

### CODE REVIEW

**Comment by JulianWielga** [comment]
there should be no changes in app behavior here, just cleanup/refactor

File: ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  groupFinish = "group-finish",
+  groupCancel = "group-cancel",
+  groupUngroup = "group-ungroup",
+  editProperties = "edit-properties",
+  processCompare = "process-compare",
+  processMigrate = "process-migrate",
+  processImport = "process-import",
+  processJSON = "process-JSON",
+  processPDF = "process-PDF",
+  processArchiveToggle = "process-archive-toggle",
```
Comment (lciolecki) [inline ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts:22]: I recognize this one.. What does it mean?

File: ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  editLayout = "edit-layout",
+  groupStart = "group-start",
+  groupFinish = "group-finish",
+  groupCancel = "group-cancel",
+  groupUngroup = "group-ungroup",
+  editProperties = "edit-properties",
+  processCompare = "process-compare",
+  processMigrate = "process-migrate",
+  processImport = "process-import",
+  processJSON = "process-JSON",
```
Comment (lciolecki) [inline ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts:20]: Let's be consistent: process-json?

File: ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  processPDF = "process-PDF",
```
Comment (lciolecki) [inline ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts:21]: Let's be consistent: process-pdf?

File: ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  processArchiveToggle = "process-archive-toggle",
+  processArchive = "process-archive",
+  processUnarchive = "process-unarchive",
+  testFromFile = "test-fromFile",
+  testGenerate = "test-generate",
+  testCounts = "test-counts",
+  testHide = "test-hide",
+  viewBussinesView = "view-bussinesView",
```
Comment (lciolecki) [inline ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts:29]: Let's be consistent: view-business? There is mistake in word.

File: ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  viewZoomIn = "view-zoomIn",
```
Comment (lciolecki) [inline ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts:30]: Let's be consistent: view-zoom-*?

**Review by lciolecki — APPROVED** [review]
LGTM, but please look at my comments.

File: ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts (hunk 3 above)
Comment (JulianWielga) [reply ui/client/components/toolbars/toolbarSettings/BuiltinButtonTypes.ts:21]: this string will be changed in last step

**Pull request merged**